ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/+(sinon.|sinon-no-sourcemaps.)*"
```

Comment:
The timers files was removed in 1.17 (see [change log](https://github.com/sinonjs/sinon/blob/v1.17/Changelog.txt). They were present up until 1.16. sinon-ie wasn't removed, only sinon-timers-ie.js. 

[This commit](https://github.com/grassator/Sinon.JS/commit/5ee643df517f6ed7296b4a1ce0a81317949526d6) shows which gives were removed, but you can also see all the names of the other files that are being generated as well.

What I don't really get is why the original glob pattern, which included all files in `pkg`, was changed to a more exclusive version. The original glob would have handled this fine. You want to distribute all the produced files, right?